app/public/cantusdata/management/commands/import_data.py

```diff
@@ -115,9 +115,7 @@ def import_manuscript_data(self, **options):
                     self.stdout.write(f"FORBIDDEN: {source_id}")
                 else:
                     self.stdout.write(f"FAILED: {source_id}")
-        self.stdout.write(
-            f"Successfully imported {i} manuscripts into database."
-        )
+        self.stdout.write(f"Successfully imported {i} manuscripts into database.")
```

Comment:
I find this one curious. I guess if it's short enough to fit on a single line, it should go on a single line, as far as Black is concerned?

No need to take any action here - I just find this interesting.

Reply:
Yeah, this one stood out to me as well....I think that is what's going on here, but it doesn't seem like this would be affected by any of the new elements of the Black style? Curious, indeed